modules/build.gradle

```diff
@@ -31,7 +31,7 @@ ext {
 		"com.liferay.gradle.plugins.css.builder.version": "5.0.9",
 		"com.liferay.gradle.plugins.theme.builder.version": "2.0.17",
 		"com.liferay.portal.tools.bundle.support.version": "3.7.4",
-		"com.liferay.portal.tools.service.builder.version": "1.0.462",
+		"com.liferay.portal.tools.service.builder.version": "1.0.463",
```

Comment:
Is upgrading service builder part of fixing this issue? If not let's send it as a separate ticket/pull.